src/components/pages/Success/Success.js

```diff
@@ -18,6 +19,10 @@ export default function Success(props) {
     history.push('/checking-buttons');
   };
 
+  setTimeout(function() {
+    window.location.replace('/checking-buttons');
```

Comment:
You have it going to the check buttons when it should be going to the qr reader instead